### DISCUSSION TITLE
URL entry for finite time

### DISCUSSION THREAD

**Discussion by WasuMrTomass0** [SinTan1729/chhoto-url#13] [Ideas]
Hello,

Is there a plan to add a "finite time" feature? 
Given entry would be enabled for (or deleted after) specific time.

Additionaly, maximum number of clicks for that link.

Example use case:
I would like to create a url for my friend to access shared files from Nextcloud app.
I would like to set endlife datetime to this url shortener so I don't have to worry about leaving this data accessible to world. (and don't have to manually delete it)

Great work! I really enjoy using your tool :)
Best regards,
Wasu

**Comment by SinTan1729** [comment]
@WasuMrTomass0 No, it's not possible. And it's unlikely to be added in the future. The goal of this project is to stay as simple as possible. For advanced use cases, there are already projects that do those.

Edit: If you want to use `chhoto-url` for this, it can be done by writing your own scripts and using something like `cron`. Take a look at the README for CLI usage instructions.

**Comment by SinTan1729** [comment]
Hi, an expiry time feature was added in v6.
